index.html

```diff
@@ -280,12 +280,12 @@ <h2>Examples</h2>
 {
   "@context": ["http://schema.org/", "https://w3id.org/security/v1"],
   "description": "Hello world!",
-  "signature": {
+  "proof": {
     "type": "Ed25519Signature2018",
     "created": "2016-10-23T05:50:16Z",
     "creator": "did:example:123456789abcdefghi",
```

Comment:
I noticed `creator` is out of date as well:

```suggestion
    "verificationMethod": "did:example:123456789abcdefghi#key1",
```